src/data_process/ttnet_data_utils.py

```diff
@@ -108,23 +108,23 @@ def get_events_infor(game_list, configs, dataset_type):
                                                                  smooth_idx + num_frames_from_event + 1)]
                 img_path_list = []
                 for sub_smooth_idx in sub_smooth_frame_indices:
-                    img_path = os.path.join(images_dir, game_name, 'img_{:06d}.jpg'.format(sub_smooth_idx))
+                    img_path = Path(images_dir) / game_name / f'img_{sub_smooth_idx:06d}.jpg'
                     img_path_list.append(img_path)
                 last_f_idx = smooth_idx + num_frames_from_event
                 # Get ball position for the last frame in the sequence
-                if '{}'.format(last_f_idx) not in ball_annos.keys():
-                    print('smooth_idx: {} - no ball position for the frame idx {}'.format(smooth_idx, last_f_idx))
+                if f'{last_f_idx}' not in ball_annos.keys():
+                    print(f'smooth_idx: {smooth_idx} - no ball position for the frame idx {last_f_idx}')
                     continue
-                ball_position_xy = ball_annos['{}'.format(last_f_idx)]
+                ball_position_xy = ball_annos[f'{last_f_idx}']
                 ball_position_xy = np.array([ball_position_xy['x'], ball_position_xy['y']], dtype=np.int)
                 # Ignore the event without ball information
                 if (ball_position_xy[0] < 0) or (ball_position_xy[1] < 0):
                     continue
 
                 # Get segmentation path for the last frame in the sequence
-                seg_path = os.path.join(annos_dir, game_name, 'segmentation_masks', '{}.png'.format(last_f_idx))
-                if not os.path.isfile(seg_path):
-                    print("smooth_idx: {} - The segmentation path {} is invalid".format(smooth_idx, seg_path))
+                seg_path = Path(annos_dir) / game_name / 'segmentation_masks' / f'{last_f_idx}.png'
+                if not Path(seg_path).is_file():
```

Comment:
`seg_path` is already a `Path` object.

I see a lot of occurrences for this, please fix all of them